domain/domain.go

```diff
@@ -1702,24 +1702,24 @@ func (do *Domain) distTaskFrameworkLoop(ctx context.Context, taskManager *storag
 		logutil.BgLogger().Info("dist task scheduler stopped")
 	}()
 
-	var dispatch dispatcher.Dispatch
+	var dispatcherPool *dispatcher.Pool
 	startDispatchIfNeeded := func() {
-		if dispatch != nil {
+		if dispatcherPool != nil {
```

Comment:
Could we not use this kind of method to check whether the dispatcherPool is inited. I thought one struct with one IsInit bool type field is better.